src/components/Search/SearchOverTime.tsx

```diff
@@ -0,0 +1,178 @@
+import { useEffect, useState, useCallback } from 'react';
+
+import InfoIcon from '@mui/icons-material/InfoOutlined';
+import Grid from '@mui/material/Grid';
+import InputLabel from '@mui/material/InputLabel';
+import Tooltip from '@mui/material/Tooltip';
+import { cssRule } from 'typestyle';
+
+import { compareView } from '../../common/constants';
+import { useAppSelector } from '../../hooks/app';
+import {
+  Spacing,
+  DropDownMenuRaw,
+  DropDownItemRaw,
+  //SearchStyles can be found in CompareCards.ts
+  SearchStyles,
+} from '../../styles';
+import { RevisionsList } from '../../types/state';
+import SearchDropdown from './SearchDropdown';
+import SearchInput from './SearchInput';
+import SearchResultsList from './SearchResultsList';
+
+interface SearchProps {
+  selectLabel: string;
+  searchResults: RevisionsList[];
+  tooltip: string;
+  inputPlaceholder: string;
+  isEditable: boolean;
+}
+
+function SearchComponent({
```

Comment:
nit: call this `SearchOverTime`

You can also directly write `export default function SearchOverTime` instead of adding the export at the end of the file. It makes it obvious that this is the exported function.

(I personally prefer named (aka non-default) exports for easier searching (because the exported and imported names are guaranteed to be the same), but because all other components are using default exports, let's keep it this way for now)

Reply:
Yes, thanks for catching that typo.